### PR TITLE
[FEAT] Husky integration

### PR DESCRIPTION
**Pull request by hv7214**
**Details**
1. Added `husky:precommit`.
2. Added prettier script.
3. precommit checks for lint and prettier errors.

**Related issue**
https://github.com/moira-alert/moira/issues/275

### CODE REVIEW

**Comment by sashasushko** [comment]
@hv7214 we have some issues with node version in Travis. So it's take some time to update.

**Comment by hv7214** [comment]
@sashasushko should i update node version in travis script?

**Comment by sashasushko** [comment]
@hv7214 no, we have devops :-) Just waiting

**Comment by hv7214** [comment]
@sashasushko build is still failing

**Comment by sashasushko** [comment]
@hv7214 it's okay. They also have weekends today (as all in Russia)

File: package.json [hunk 1]
```diff
@@ -10,14 +10,21 @@
         "fakeapi": "json-server --watch fakeapi/db.json --routes fakeapi/routes.json --port 9002",
         "start": "npm-run-all --parallel serve fakeapi",
         "lint": "eslint --ext .jsx --ext .js src/",
-        "test": "cross-env NODE_ENV=testing jest \"./src\""
+        "test": "cross-env NODE_ENV=testing jest \"./src\"",
+        "prettier": "prettier \"**/*.js\""
```
Comment (sashasushko) [inline package.json]: Do you check this line? Run it locally? Not need prettify only files in `src` directory?

Comment (hv7214) [reply package.json]: Sorry i didn't get it. Btw i've tested prettier script. Do you mean to constrict the scope of prettier to `src` folder!

Comment (sashasushko) [reply package.json]: Yep. I think it's unnecessary to check `webpack.dev.js` or other scripts in root folder

Comment (hv7214) [reply package.json]: yeah you are right :+1:

**Pull request merged**